sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 4.2.0-beta.3 (Unreleased)
-
+## 4.2.0-beta.3 (2020-06-10)
+- Updated version for azure-core, azure-identity and external dependencies.
+- Fixed `ByteBuff` resource leak in `KeyVaultCredentialPolicy`.
```

Comment:
Was this leaking `ByteBuff` or `ByteBuffer`?

Reply:
It's `ByteBuf`. I'll correct the CHANGELOG :)